Glob/en-US/about_Glob.help.txt

```diff
@@ -0,0 +1,49 @@
+TOPIC
+    about_Glob
+
+
+DESCRIPTION
+    Glob is a PowerShell module that lets you find files using advanced glob 
+    syntax. It gives you greater control over files returned in recursive 
+    searches. You can use the standard `*`, `?`, and `[]` wildcard operators 
+    to match file/directory names. You can also use the `**` operator to 
+    match multiple directory names.
+    
+    For example, the pattern `**\*.ps1` would return all `*.ps1` files in the 
+    current and all sub-directories.
+
+    Glob only exposes one function: `Find-GlobFile`. Once you've installed 
+    and loaded the Glob module, run `help Find-GlobFile` for more information 
+    and examples.
+
+
+SYSTEM REQUIREMENTS
+
+    * Windows PowerShell 4+ running under .NET Framework 4.6 or later
+    * PowerShell Core 6+ running under .NET Core.

```

Comment:
Did a quick manual test of this module on PowerShell Core 6.1.1 running on Ubuntu 18.04 and I got unexpected results from `Find-GlobFile`. The main thing I noticed was that when searching for files in the current root directory I always needed to preface the pattern with `./`, e.g. the command `Find-GlobFile -Path '/etc' -Include './*.conf'` finds the same files as `ls -l *.conf`, but using the pattern `-Include '*.conf'` returns nothing.

Not sure if this is a bug or something deeper with within PowerShell Core or the DotNet Globbing library, but I wanted to point it out since we document that this module works  on `PowerShell Core 6+`. Maybe we need a `* Tested on .NET Core on Windows only` note?

Reply:
Whiskey's Pester task fails when run under PowerShell Core. We need to fix that before I can run these tests under PowerShell Core. I'll leave this in for now since a workaround exists.

Reply:
Right, I more meant that it'd be more accurate for us to document this as:

> Tested with PowerShell Core 6+ <sup>* on Windows only</sup>

Pedantic, I know, but just pointing it out for posterity.